detector/aws_instance_previous_type.go

```diff
@@ -6,7 +6,15 @@ import (
 	"github.com/wata727/tflint/issue"
 )
 
-func (d *Detector) DetectAwsInstancePreviousType(issues *[]*issue.Issue) {
+type AwsInstancePreviousTypeDetector struct {
+	*Detector
+}
+
+func (d *Detector) CreateAwsInstancePreviousTypeDetector() *AwsInstancePreviousTypeDetector {
```

Comment:
__[golint]__
* [golint] exported method Detector.CreateAwsInstancePreviousTypeDetector should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->